stages/org.osbuild.tuned

```diff
@@ -33,8 +33,14 @@ class TunedProfilesDB:
         profile_config_file = "tuned.conf"
 
         for path in profile_directories:
-            for dir_entry in os.scandir(f"{tree}{path}"):
-                if dir_entry.is_dir() and os.path.isfile(f"{tree}{path}/{dir_entry.name}/{profile_config_file}"):
+            profiles_dir = f"{tree}{path}"
+            # Since version 2.23.0, TuneD by default uses `profiles/` subdirectory.
+            # If the subdirectory exists, we look for profiles there.
+            if os.path.isdir(f"{profiles_dir}/profiles"):
+                profiles_dir = f"{profiles_dir}/profiles"
+
+            for dir_entry in os.scandir(profiles_dir):
+                if dir_entry.is_dir() and os.path.isfile(f"{profiles_dir}/{dir_entry.name}/{profile_config_file}"):
```

Comment:
I generally prefer using `os.path.join` for concatenations like this; but not blocking.

Reply:
While I agree, I have two poor reasons to keep it as is:

1. I wanted to maintain consistency in the file. Some concatenated paths are all absolute (not on this line), which means additional handling when using `os.path.join()`. While entirely doable, it would increase verbosity.
2. This specific line would get longer than 120 characters if I used `os.path.join()`, meaning splitting it and subjectively making it harder to read. :innocent: 

If others would like it more, I can rework the code to use `os.path.join()`.